qgis_deployment_toolbelt/utils/computer_environment.py

```diff
@@ -29,6 +31,22 @@
 # ##################################
 
 
+def datetime_dict() -> dict:
```

Comment:
To keep things consistent, couldn't we call it `date_dict()`?

For `environment_dict()` in the json we have `$.environment` and for `datetime_dict() `we have `$.date`